content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/270-baseline-your-database.mdx

```diff
@@ -0,0 +1,397 @@
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['node', 'planetscale']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-node-planetscale"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-node-planetscale"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['node', 'sqlserver']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-node-sqlserver"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-node-sqlserver"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['node', 'cockroachdb']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-node-cockroachdb"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-node-cockroachdb"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['typescript', 'postgresql']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-typescript-postgres"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-typescript-postgres"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['typescript', 'mysql']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-typescript-mysql"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-typescript-mysql"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['typescript', 'sqlserver']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-typescript-sqlserver"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-typescript-sqlserver"
+  arrow
+>
+  Next Steps
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['typescript', 'cockroachdb']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./install-prisma-client-typescript-cockroachdb"
+  arrowLeft
+>
+  Install Prisma Client
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./next-steps-typescript-cockroachdb"
```

Comment:
If this comes before "next steps", should this not also trigger a change on the "next steps" page back?

Reply:
Copy-paste mistake. 🙈

I'll fix it